examples/third_person_control/generate_actions.py

```diff
@@ -0,0 +1,45 @@
+import argparse
```

Comment:
I thought we agreed that we didn't need to generate actions ahead of time like this. I like the sequential for loops in the car example.

Reply:
removed.